### DISCUSSION TITLE
How to get Student Developer Pack?

### DISCUSSION THREAD

**Discussion by PavloVasylchenko** [community/community#39007] [GitHub Education]
Hello

**Comment by dhrubasaha08** [comment]
try using a different mail id

**Comment by samus-aran** [comment]
Hi @PavloVasylchenko 👋 

GitHub does support remote learners so I suspect that the rejection reason could be that you need to modify your browser settings for GitHub to use your location information. By not doing so we are unable to accurately assess the application.

Cancel your current application (if you have one) and re-apply. You should receive a prompt to enable location access when you try to proceed to the next page of the application.

Let us know how you get on 👍
